### ISSUE TITLE
Ask clients to pay for scope if it includes domain research?

### ISSUE THREAD

**Issue by RachBLondon** [foundersandcoders/mvps#3]
It's just an idea, but maybe if a client comes with a project that requires quite a bit of research. (particularly if its domain research that really the client should have already prepared), we could possibly ask them to pay a fee (equivalent to a day or so) to scope the project?

For example if I client wanted to use a specific API but hadnt researched it fully (and the legal implications) maybe we could charge them.

**Comment by sofer** [comment]
@RachBLondon Interesting point. We could certainly do that.

**Comment by nelsonic** [comment]
@RachBLondon absolutely. if you are doing *any* work for a client it should be tracked so they can be billed. the work you (and your team) did for the flight delays chap should definitely be charged.

**Comment by Jbarget** [comment]
@nelsonic how do you tend to 'track' your research work?

**Comment by nelsonic** [comment]
@Jbarget write down *everything* in a Readme (*or Google Doc if "sensitive"*)
and try to track the number of minutes/hours/days you spend on it.
Its not an *exact* measure, but its *essential* otherwise clients/managers are inclined to think that (*everything*) takes "*5 minutes*" ...

**Comment by Jbarget** [comment]
@nelsonic gotcha, thanks
